tests/fgcmcalTestBase.py

```diff
@@ -92,16 +88,18 @@ def _runPipeline(self, repo, pipelineFile, queryString=None,
             Pipeline definition file.
         queryString : `str`, optional
             String to use for "-d" data query.
```

Comment:
This should probably be "where query that defines the data to use" or something like that, since `-d` is no longer relevant.